scripts/import_gff3.py

```diff
@@ -0,0 +1,347 @@
+        gff_cols = pd.read_table(filename, sep='\t', skiprows=3, header=None).columns
+         
+        # Adquiring non-attributes data
+        body_data=pd.read_table(filename, sep='\t', skiprows=3, header=None, usecols=gff_cols[0:-1])
+        body_data.columns = ['SeqID', 'source', 'type', 'start', 'end', 'score', 'strand', 'phase']
+        # Adquiring attributes data
+        atr_data = pd.read_table(filename, sep='\t', skiprows=3, header=None, usecols=gff_cols[[-1]])
+    
+        # Splitting the attributes column
+        list_atr = []
+        # cheking attributes present in first row
+        attr_names=atr_data.values[0,0].split()[0::2]      #attributes in the column
+        num_attr = len(attr_names)                         #number of attributes
+        expression_colindex=attr_names.index ('Expression')  #position of the expression column in the attr column 
+        if verbose==True:
+            print num_attr,' attributes in the file '
+            print '--------------------------------------'
+            for attr in attr_names:
+                print attr
+            print '--------------------------------------'
+        # joining rows of attributes without the descriptor
+        for row in range(atr_data.shape[0]):
+            list_atr.append(atr_data.values[row, 0].split()[1::2])
+    
+        # appending observations
+        atr_data = pd.DataFrame(list_atr, columns=attr_names)
+        # Eliminating ;
+        for i in attr_names:
+            atr_data[i] = atr_data[i].replace({';': ''}, regex=True)
+    
+        # desglosing the expression column in a column for each sample
+        list_expression=[]
+        for row in range(atr_data.shape[0]):
+            list_expression.append(atr_data.values[row,expression_colindex].split(','))
+        sample_names=['Expression_' +x for x in sample_names]
+        
+        expression_data = pd.DataFrame(list_expression, columns=sample_names)
+        
+        atr_data=atr_data.drop('Expression',axis=1) #Remove the expression column
+        atr_data=atr_data.join(expression_data)
+    
+        # Joining the body and attributes dataframes
+        data = body_data.join(atr_data)
+    
+        return data
+    except:
+        print 'Error loading the file'
+
+
+"""
+Function that check the header then load a gff file and check the content
+Returns the dataframe if the format is ok, false if not
+"""
+def load_gff3(filename):
+    try:
+        Error = False     
+        # Checking the format file 
+        # Header and 1st data row
+        header_rows=0
+        with open(filename) as file:
+            rowfile=file.readline()
+            header_rows=header_rows+1
+
+            while True:
+                if rowfile.startswith('#'):
+                    rowfile=file.readline()
+                    header_rows=header_rows+1
+                else:
+                    data_1=rowfile.split('\t')
+
+                    if header_rows!=4:
```

Comment:
It can be any number of lines. Just check that the lines starts with "## " always in the header